graphql/resolvers.go

```diff
@@ -3143,14 +3146,34 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 			if err != nil {
 				return false, err
 			}
+			projectMatches := false
 			for _, projectName := range perfPlugin.Projects {
 				if projectName == pRef.Id || projectName == pRef.Identifier {
-					return true, nil
+					projectMatches = true
+					break
 				}
 			}
+			if !projectMatches {
+				return false, nil
+			}
+		}
+		opts := apimodels.GetCedarPerfCountOptions{
+			BaseURL:   evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+			TaskID:    utility.FromStringPtr(obj.Id),
+			Execution: obj.Execution,
+		}
+		if opts.BaseURL == "" {
+			return false, nil
+		}
```

Comment:
nit. We can move this check to the top of the file. And avoid making any queries if `evergreen.GetEnvironment().Settings().Cedar.BaseURL == ""`